### ISSUE TITLE
Provide sample xml and certificate file

### ISSUE THREAD

**Issue by vaishnavkishan** [blocksterlabs/aadhaar-offlinexml-verifier#1]
Please provide sample data with the program.

I can not get the algorithm working with the certificate provided by the UIDAI and the sample file is not downloading for some reason.

**Comment by jaiakash** [comment]
Yeah, also can we some documentation how to setup and run the project?
